src/TorSharp/TorSharp.csproj

```diff
@@ -3,10 +3,11 @@
   <PropertyGroup>
     <TargetFrameworks Condition="'$(CoreOnly)' != 'true'">net462;net472;netstandard2.0</TargetFrameworks>
     <TargetFrameworks Condition="'$(CoreOnly)' == 'true'">netstandard2.0</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
     <AssemblyName>Knapcode.TorSharp</AssemblyName>
     <RootNamespace>Knapcode.TorSharp</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <NoWarn>1591</NoWarn>
+    <NoWarn>1591;S2346</NoWarn>
```

Comment:
What code is `S2346`?

Reply:
Oh it's SonarLint analyzer rules.
https://rules.sonarsource.com/csharp/RSPEC-2346

Reply:
I'm fine with having it there if you want it. It seems a bit odd to have a command there for a non-default analyzer, or an analyzer that isn't pulled in as part of the repo.